src/components/Section/shared/SectionNavigation.jsx

```diff
@@ -56,19 +25,37 @@ const SectionNavigation = ({ section, subsection, formType }) => {
         <div className="btn-container">
           <SectionNavButton
             direction="back"
-            label={backLabel}
+            label={back && back.navLabel}
             onClick={backOnClick}
-            isEmpty={!back} />
+            isEmpty={!back}
+          />
           <div className="btn-spacer" />
           <SectionNavButton
             direction="next"
-            label={nextLabel}
+            label={next && next.navLabel}
             onClick={nextOnClick}
-            isEmpty={!next} />
+            isEmpty={!next}
+          />
         </div>
       </div>
     </div>
   )
 }
 
-export default SectionNavigation
+/* eslint react/forbid-prop-types: 0 */
+SectionNavigation.propTypes = {
+  back: PropTypes.object,
+  next: PropTypes.object,
+}
+
+SectionNavigation.defaultProps = {
+  back: null,
+  next: null,
+}
+
+const mapStateToProps = (state, ownProps) => {
+  const { section, subsection } = ownProps
+  return getBackAndNext(state, { section, subsection })
```

Comment:
This fixed the tests that use a mockStore with initial values -- basically, it seems like selectors should be used when connecting the store to a component (instead of the selector importing the store instance directly), which makes sense!